Include/internal/pycore_critical_section.h

```diff
@@ -53,13 +53,13 @@ extern "C" {
 
 // Asserts that the mutex is locked.  The mutex must be held by the
 // top-most critical section otherwise there's the possibility
-// that the mutex would be swalled out in some code paths.
+// that the mutex would be stalled out in some code paths.
```

Comment:
Not sure what `swalled` is supposed to mean. I can just revert this change and the same one below if you want.

Reply:
@DinoV wrote this.

Refuse the temptation to guess unless you know what it means.

Having said that, my best guess is “swapped out”. The LL and PP keys are in the same corner of my keyboard.